bundle/src/projects/commercial/modules/dfp/render-advert-label.ts

```diff
@@ -71,32 +71,33 @@ const createAdTestLabel = (
 	return adTestLabel;
 };
 
-const createAdTestCookieRemovalLink = (
-	adTestName: string | null,
-): HTMLElement => {
+const createAdTestCookieRemovalLink = (): HTMLElement => {
 	const adTestCookieRemovalLink = document.createElement('div');
 	adTestCookieRemovalLink.style.cssText =
-		'position: relative;padding: 0 0.5rem;text-align: left;box-sizing: border-box;';
+		'position: relative;padding: 0;text-align: left;box-sizing: border-box;display: block:width: 0;height: 0';
 
-	if (adTestName) {
-		const url = new URL(window.location.href);
-		url.searchParams.set('adtest', 'clear');
+	const url = new URL(window.location.href);
+	url.searchParams.set('adtest', 'clear');
 
-		const clearLink = document.createElement('a');
-		clearLink.className = 'ad-slot__adtest-cookie-clear-link';
-		clearLink.href = url.href;
-		clearLink.innerHTML = 'clear';
+	const clearLink = document.createElement('a');
+	clearLink.className = 'ad-slot__adtest-cookie-clear-link';
+	clearLink.href = url.href;
+	clearLink.innerHTML = 'clear';
 
-		adTestCookieRemovalLink.appendChild(clearLink);
-	}
+	adTestCookieRemovalLink.appendChild(clearLink);
 
 	return adTestCookieRemovalLink;
 };
 
 const renderAdvertLabel = (adSlotNode: HTMLElement): Promise<Promise<void>> => {
 	return fastdom.measure(() => {
 		if (shouldRenderLabel(adSlotNode)) {
-			const renderAdTestLabel = shouldRenderAdTestLabel();
+			const renderAdTestLabel = shouldRenderAdTestLabel(adSlotNode);
+			const adTestClearExists =
+				adSlotNode.parentNode?.firstElementChild?.firstElementChild
+					?.nodeName === 'A'
+					? true
+					: false;
```

Comment:
Is this checking if the elment is an anchor tag? If so, it might be more idiomatic to use

```typescript
element instanceof HTMLAnchorElement
```

Reply:
Oh nice idea! I've made the change